generate/templates/templates/binding.gyp

```diff
@@ -1,6 +1,7 @@
 {
   "variables": {
-    "is_electron%": "<!(node ./utils/isBuildingForElectron.js <(node_root_dir))"
+    "is_electron%": "<!(node ./utils/isBuildingForElectron.js <(node_root_dir))",
+    "is_IBMi%": "<!(node ./utils/isBuildingForIBMi.js)"
```

Comment:
`"is_IBMi%": "OS == 'aix' and <!(node -e "console.log(os.type())) == 'OS400'"`

Reply:
That code doesn't seem to work. It seems to assign the string "OS == 'aix' and OS400 == 'OS400' to the variable. The simplest way I found to do it is like this:

`is_IBMi%": "<!(node -p \"os.platform() == 'aix' && os.type() == 'OS400' ? 1 : 0\")"`

Would that be acceptable?

Reply:
Also, no, unfortunately I'm not aware of any CI tools for test / build on IBM i.

Reply:
Yes that would be acceptable.